modules/admin-ui-picard/app/src/reducers/seriesDetailsReducers.js

```diff
@@ -0,0 +1,98 @@
+/**
+ * This file contains redux reducer for actions affecting the state of a series
+ */
+import {
+    LOAD_SERIES_DETAILS_ACLS_SUCCESS,
+    LOAD_SERIES_DETAILS_FAILURE,
+    LOAD_SERIES_DETAILS_FEEDS_SUCCESS,
+    LOAD_SERIES_DETAILS_IN_PROGRESS,
+    LOAD_SERIES_DETAILS_METADATA_SUCCESS,
+    LOAD_SERIES_DETAILS_THEME_NAMES_FAILURE, LOAD_SERIES_DETAILS_THEME_NAMES_IN_PROGRESS,
+    LOAD_SERIES_DETAILS_THEME_NAMES_SUCCESS,
+    LOAD_SERIES_DETAILS_THEME_SUCCESS,
+} from "../actions/seriesDetailsActions";
+
+// Initial state of series details in redux store
+const initialState = {
+    isLoading: false,
+    metadata: {},
+    feeds: {},
+    acl: {},
+    theme: '',
+    themeNames: []
+};
+
+// Reducer for series details
+const seriesDetails = (state=initialState, action) => {
+    const { type, payload } = action;
+    switch (type) {
+        case LOAD_SERIES_DETAILS_IN_PROGRESS: {
+            return {
+                ...state,
+                isLoading: true
+            }
+        }
+        case LOAD_SERIES_DETAILS_METADATA_SUCCESS: {
+            const { seriesMetadata } = payload;
+            return {
+                ...state,
+                isLoading: false,
```

Comment:
This is used in the thunk without a `LOAD_SERIES_DETAILS_IN_PROGRESS` or `LOAD_SERIES_DETAILS_FAILURE`. I think in that case the `isLoading: false` shouldn't be here